src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { PostInfo } from '../PostInfo/PostInfo';
+import { Post } from '../../types/Post';
+
+import './PostList.scss';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <li className="PostList__item">
+        <PostInfo key={post.id} post={post} />
+      </li>
```

Comment:
The key attribute should be on the first wrapper element )

```suggestion
      <li key={post.id} className="PostList__item">
        <PostInfo post={post} />
      </li>
```